docs/wiki/Deploying-ALZ-VWAN.md

```diff
@@ -69,12 +69,38 @@ Depending on your requirements, you may choose to deploy additional network infr
 
 ![vwan](./media/clip_image078.jpg)
 
+### Deploying networking resources in an additional region
+
+You may also deploy an additional virtual hub in an additional region, to prepare for multi-region operations.
```

Comment:
should we be stronger in our wording of this being our recommended approach - i personally think so

Reply:
Updated in the commit, let me know your thoughts!